sql/core/benchmarks/JSONBenchmarks-results.txt

```diff
@@ -0,0 +1,33 @@
+================================================================================================
+Benchmark for performance of JSON parsing
+================================================================================================
+
+OpenJDK 64-Bit Server VM 1.8.0_163-b01 on Windows 7 6.1
+Intel64 Family 6 Model 94 Stepping 3, GenuineIntel
+JSON schema inferring:                   Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+No encoding                                 48088 / 48180          2.1         480.9       1.0X
+UTF-8 is set                                71881 / 71992          1.4         718.8       0.7X
+
+OpenJDK 64-Bit Server VM 1.8.0_163-b01 on Windows 7 6.1
+Intel64 Family 6 Model 94 Stepping 3, GenuineIntel
+JSON per-line parsing:                   Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+No encoding                                 12107 / 12246          8.3         121.1       1.0X
+UTF-8 is set                                12375 / 12475          8.1         123.8       1.0X
```

Comment:
Hi, @HyukjinKwon . According to the ratio, it seems to be a regression on `No encoding` case. How do you think this change?

Reply:
Wait .. this is almost 50% slower. This had to be around 8000ish.

Reply:
I also run this benchmark and got the same ratio. So it's a little weird.
- https://github.com/heary-cao/spark/pull/3/files#diff-7676fb48b895486092bea2fb491e6de4R18

Reply:
IIRC, this benchmark was added rather we can make sure setting encoding does not affect the performance without encoding (right @MaxGekk ?). We should fix this. @cloud-fan

Reply:
Let me take a quick look within few days. This is per line basic case where many users are affected.

Reply:
Ah, I see. This is also because of count optimization. ratio is weird but actually it's performance improvement for both cases. shouldn't be a big deal.

Reply:
Yup, it's by https://github.com/apache/spark/commit/a8a1ac01c4732f8a738b973c8486514cd88bf99b

Before:

```
JSON per-line parsing:                   Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
------------------------------------------------------------------------------------------------
No encoding                                 35786 / 36446          2.8         357.9       1.0X
UTF-8 is set                                57486 / 58714          1.7         574.9       0.6X
```

After:


```
JSON per-line parsing:                   Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
------------------------------------------------------------------------------------------------
No encoding                                 11142 / 11425          9.0         111.4       1.0X
UTF-8 is set                                11139 / 11293          9.0         111.4       1.0X
```

Looks not a regression.

Reply:
Thank you for the confirmation!